config/environments/test.rb

```diff
@@ -54,7 +54,7 @@
 
   # Configuration to stop Cost of Living landing page showing in production before
   # campaign goes live
-  config.unreleased_features = true
+  config.unreleased_features = false
```

Comment:
This is not a no blocker. However, I would set this value to `true`. If you do that you'll have less clean to do in the tests when this page goes live.